SmartDeviceLinkTests/ProxySpecs/SDLSecondaryTransportManagerSpec.m

```diff
@@ -700,6 +750,7 @@ + (void)swapConnectionMethods {
                 manager.secondaryTransportType = SDLTransportSelectionTCP;
                 manager.ipAddress = @"192.168.1.1";
                 manager.tcpPort = 12345;
+                manager.appReady = YES;
```

Comment:
When the transport is closed, `appReady` is going to be `NO`, right? If not, then when you enter the `reconnecting` state in the `SecondaryTransportManager` you need to reset it to `NO`.